src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -99,16 +100,19 @@ def acs_browse(resource_group, name, disable_browser=False):
     :type resource_group_name: String
     :param disable_browser: If true, don't launch a web browser after estabilishing the proxy
     :type disable_browser: bool
+    :param ssh_key_file: If set a path to an SSH key to use, only applies to DCOS [optional]
```

Comment:
`[optional]` is redundant as unless a param is marked as `[Required]` it is optional.